first draft of primary page (#4437)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary.md

---
title: Become a primary school teacher
subject: primary
title_paragraph: |-
  <p>

>   As a primary school teacher, you have the opportunity to foster a love of learning in pupils from the very beginning. You'll create a supportive and vibrant learning environment for pupils' first encounters with new ideas and give them precious 'aha' moments.</p>

Reviewer: ```suggestion
As a primary school teacher, you'll have the opportunity to foster a love of learning in pupils from the very beginning. You'll create a supportive and vibrant classroom environment where pupils can experience the joy of learning.</p>
```

Reviewer: The above might be a bit over the top, so feel free to reword!

description: |-

>     Find out how to become a chemistry teacher, including what you'll teach and if it is the right career for you.

Reviewer: ```suggestion
    Find out how to become a primary school teacher, including what you'll teach and if it is the right career for you.
```
